test/lib/ansible_test/_util/target/sanity/import/importer.py

```diff
@@ -149,10 +157,32 @@ def __init__(self, path, name, restrict_to_module_paths):
             self.loaded_modules = set()
             self.restrict_to_module_paths = restrict_to_module_paths
 
+        def find_spec(self, fullname, path=None, target=None):  # pylint: disable=unused-argument
+            """Return the spec from the loader or None
+            :param fullname: str
+            :param path: List[str]
+            :param target: str
+            :return: ModuleSpec | None
+            """
+            loader = self._get_loader(fullname, path=path)
+            if loader is not None:
+                if has_py3_loader:
+                    return spec_from_loader(fullname, loader)
+                raise ImportError("Failed to import '%s' due to a bug in ansible-test. Check importlib imports for typos." % fullname)
```

Comment:
I'm not clear on what conditions would trigger this, or what the user's action should be if it did occur.